components/Common/Button/Button.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { MouseEventHandler } from 'react';
+import { ButtonContainer } from './Button.styles';
+
+export interface ButtonProps {
+  children: React.ReactNode;
+  hasShadow?: boolean;
+  size?: 'lg' | 'md';
+  color?: 'primary' | 'gray';
```

Comment:
```suggestion
  hasShadow: boolean;
  size: 'lg' | 'md';
  color: 'primary' | 'gray';
```

default값을 넣어줬으니 옵셔널은 제거해주는게 더 좋아보입니다!

Reply:
Button.styles.ts 에서 에러가 나서 optional property 로 했는데 다시 제거해보니 정상적으로 되네요!
[b778dd1](https://github.com/depromeet/11th_5team_fe/pull/39/commits/b778dd196181c56f267fdab1416013aaf4c603dd) 수정했습니다!